include/swift/AST/DiagnosticsParse.def

```diff
@@ -1616,6 +1616,8 @@ ERROR(attr_private_import_expected_sourcefile_name,none,
       "expected a source file name in @_private(sourceFile:)", ())
 ERROR(attr_private_import_expected_colon,none,
       "expected ':' after @_private(sourceFile", ())
+ERROR(operator_in_import_path,none,
```

Comment:
Maybe something like `unexpected_operator_in_import_decl`? Would not group this here with diagnostics for the underscored `@_private` import feature but with other diagnostics for regular import declarations.

Reply:
I like that, I settled on `unexpected_operator_in_import_path` and I've moved the diagnostic to more similar ones in my newest commit